clients/tests/src/main/java/org/projectnessie/client/tests/AbstractSparkTest.java

```diff
@@ -49,6 +49,8 @@
 
   protected static NessieClient nessieClient;
 
+  protected abstract void initNessieClient();
```

Comment:
this should return the `NessieClient` and not let the implementations update the field in this class IMO

Reply:
the new mechanism also creates a new `NessieClient` instance for each test, but never closes them. either create only one instance or close the instances after each test (I'd prefer one instance per class)